mautic/mautic
# [DPMMA-1443] Email report multisieries piechart

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | 🔴 
| New feature/enhancement? (use the a.x branch)      | 🟢
| Deprecations?                          | 🔴
| BC breaks? (use the c.x branch)        | 🔴
| Automated tests included?              | 🟢 
| Related user documentation PR URL      | mautic/user-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation-new#... 
| Issue(s) addressed                     | Fixes #... 



## Description

Adding a Pie chart to the raport with e-mail data: sent, read, clicked, unsubscribed.

![image](https://github.com/user-attachments/assets/564793fb-bdba-405f-ad61-1123a240c6b7)









---
### 📋 Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))

2. Create and send several emails
3. Open and Read sent emails, click some links
4. Create a report with data source emails, and Graph Sent / Read / Clicked / Unsubscribed (Pie chart)
5. Check the results
